imap-simple/imap-simple.d.ts

```diff
@@ -0,0 +1,69 @@
+// Type definitions for imap-simple v1.6.3
+// Project: https://github.com/chadxz/imap-simple
+// Definitions by: Jeffery Grajkowski <https://github.com/pushplay/>
+// Definitions: https://github.com/psnider/DefinitelyTyped/imap-simple
+
+/// <reference path="../imap/imap.d.ts" />
+
+declare namespace IMAPS {
+
+    export interface ImapSimpleOptions {
+        /** Options to pass to node-imap constructor. */
+        imap: IMAP.Config;
+
+        /** Time in milliseconds to wait before giving up on a connection attempt. (Deprecated: please use options.imap.authTimeout instead) */
+        connectTimeout?: number;
+    }
+
+    export interface MessageBodyPart extends IMAP.ImapMessageBodyInfo {
+        /** string type where which=='TEXT', complex Object where which=='HEADER' */
+        body: any;
```

Comment:
That would be a string or an empty object, and it's definitely not empty.  There may also be other values for `which`.  I know very little about the IMAP spec.